osm-for-cities/values.production.yaml

```diff
@@ -47,6 +47,7 @@ runner:
     TMP_DIR: /home/runner/app/app-data/tmp
     GITEA_USER: runner
     GITEA_EMAIL: runner@osmforcities.org
+    GIT_HISTORY_START_DATE: "2022-01-01T23:59:59.999Z"
```

Comment:
@Rub21 By default, the runner will start the git history 10 days before the first run. If we set to 2022 it will take a long time to catch up. I think we can remove `GIT_HISTORY_START_DATE` from this file and from staging.yml.